fluss-rpc/src/main/java/org/apache/fluss/rpc/protocol/ApiKeys.java

```diff
@@ -79,7 +79,8 @@ public enum ApiKeys {
     REMOVE_SERVER_TAG(1048, 0, 0, PUBLIC),
     REBALANCE(1049, 0, 0, PUBLIC),
     LIST_REBALANCE_PROGRESS(1050, 0, 0, PUBLIC),
-    CANCEL_REBALANCE(1051, 0, 0, PUBLIC);
+    CANCEL_REBALANCE(1051, 0, 0, PUBLIC),
+    PRE_LAKE_TABLE_SNAPSHOT(1052, 0, 0, PRIVATE);
```

Comment:
`PRE_LAKE_TABLE_SNAPSHOT` -> `PREPARE_LAKE_TABLE_SNAPSHOT`

While `PRE` might imply "before," it’s ambiguous and non-idiomatic in event or phase naming. `PREPARE` clearly conveys the intent.